espnet2/tasks/abs_task.py

```diff
@@ -577,6 +577,24 @@ class ArgumentDefaultsRawTextHelpFormatter(
             default=None,
             help="Specify wandb id",
         )
+        group.add_argument(
+            "--wandb_entity",
+            type=str,
+            default=None,
+            help="Specify wandb entity",
+        )
+        group.add_argument(
+            "--wandb_name",
+            type=str,
+            default=None,
+            help="Specify wandb run name",
+        )
+        group.add_argument(
+            "--model_log_interval",
```

Comment:
Could you change --model_log_interval to --wandb_--model_log_interval? I'd like to unify these options to --wandb_*.

Reply:
okay. makes sense. I'll change it to `wandb_model_log_interval`